### PR TITLE
added migration files

### PR DESCRIPTION
**Pull request by sanjeevbhu22**

### CODE REVIEW

File: src/db/migrations/20231129122640_init.js
Comment (abhashkmr) [inline src/db/migrations/20231129122640_init.js:1]: use only typescript files

Comment (sanjeevbhu22) [reply src/db/migrations/20231129122640_init.js:1]: ok sir

Comment (sanjeevbhu22) [reply src/db/migrations/20231129122640_init.js:1]: done sir

**Comment by abhashkmr** [comment]
no need

**Pull request closed without merging**